### DISCUSSION TITLE
我遇到的一个离谱bug

### DISCUSSION THREAD

**Discussion by jh0072110** [immortalwrt/homeproxy#83] [General]
homeproxy在运行一阵（不到一小时）后，会导致pppoe掉线且无法再链接（必须重启光猫）
是不是非常离谱……
我也不知道是不是我自己遇到的bug，但能够100%复现……
master分支immortalwrt

**Comment by 1715173329** [comment]
推荐上街找算命先生帮你算一卦。

**Reply by jh0072110** [reply]
天灵大佬您说这bug…我甚至都不知道该给您什么日志文件……哈哈哈……
